pep-0572.rst

```diff
@@ -615,12 +615,36 @@ Broadly the same semantics as the current proposal, but spelled differently.
 
        stuff = [[f(x) as y, x/y] for x in range(5)]
 
-   Since ``EXPR as NAME`` already has meaning in ``except`` and ``with``
-   statements (with different semantics), this would create unnecessary
-   confusion or require special-casing (eg to forbid assignment within the
-   headers of these statements).
+   Since ``EXPR as NAME`` already has meaning in ``import``,
+   ``except`` and ``with`` statements (with different semantics), this
+   would create unnecessary confusion or require special-casing
+   (e.g. to forbid assignment within the headers of these statements).
 
-   TODO: Explain more why this is bad, since this keeps coming up (it's also readability)
+   Additional reasons to prefer ``:=`` over this spelling include:
+
+   - In ``if f(x) as y`` the assignment target doesn't jump out at you
+     -- it just reads like ``if f x blah blah`` and it is too similar
+     visually to ``if f(x) and y``.
```

Comment:
Huh!  I wrote a longish comment here which appears to have wholly vanished ☹️ 

Didn't ask for any changes.  Short course:  many complained that the PEP was destroying that Python "reads like English", as if punctuation didn't play a vital role in written English too.  Your point here is, to me, very strong.  `:=` prevents the text from looking like "uniform identifier salad", driving the eye to that something "important" is going on.

Just FYI, while Nick's `given` reads better to me in _some_ cases, one of its advocates  (not Nick) really soured me on it by trying to sell this as an "excellent" use:

    if some_condition and stuff.lower().strip().replace('-', '') == existing_value given stuff = get_user_input():

"Uniform identifier salad"!  I had to scan the line 3 times before I even noticed the crucial `given`, which, because of the underscore, "looks like" just the third of four identifiers in a row.  It's a Good Thing that `:=` doesn't look like an identifier!